doc/services/shell/index.rst

```diff
@@ -97,6 +99,86 @@ considerably. For that cost, it will enable the line editing,
 `tab completion <tab-feature_>`_, and `history <history-feature_>`_
 features of the shell.
 
+USB CDC ACM Backend
+===================
+
+To configure Shell USB CDC ACM backend:
+
+* Enable :kconfig:option:`CONFIG_SHELL_BACKEND_SERIAL`, along with other USB
```

Comment:
I thought we had a snippet?

Reply:
If I understand it correctly, the snippet cdc-acm-console is missing the zephyr,shell-uart redirection to the appropriate node.

I can file a separate PR to have that extra line included in the snippet.

LMk if I'm missing something!

Reply:
ah alright, I was thinking of the console then. disregard the comment

Reply:
Integrated now!